webpack/assets/javascripts/react_app/components/common/Alert.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Icon from './Icon';
+import { ALERT_CSS } from '../../constants';
+
+const Alert = (props) => {
+  let classNames = ALERT_CSS[props.type];
+
+  if (props.css) {
+    classNames += ' ' + props.css;
+  }
+
+  if (props.dismissable) {
+    classNames += ' ' + ALERT_CSS.dismissable;
+  }
+
+  const closeBtn = (
+    <button className="close" data-dismiss="alert" aria-hidden="true">
+      <Icon type="close" />
```

Comment:
We shouldn't use the bootstrap js. It manipulates the DOM. Need to be careful about inconsistencies between the DOM, the virtual DOM and the data.